CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 
 ### 💡 Enhancements 💡
 
+- Updated Batch Processor to remove unnecessary cast to `Sizer` interface (#5937)
```

Comment:
Remove this, no need will skip the changelog.

Reply:
Done